app/assets/js/app.scss

```diff
@@ -25,17 +25,18 @@
 		background-color: var(--app-lightbox-background-color);
 		z-index: $app-lightbox-z-index;
 		bottom: 0;
```

Comment:
What about bottom?

Reply:
I wanted to start refactoring all the `top`, `right`, `bottom` and `left` properties to `inset-*`. But this PR is becoming huge (76 files), so I separated the work to several tasks.

@arielk If your prefer to do this in a single PR, I can add the changes to this PR. Your call.